Information/Design.md

```diff
@@ -89,3 +89,320 @@
     - Summary: The documentation shows this REST API should provide all the features we require 
       and is publicly accessible. Using the already provided API and hosting, this option would require less resources 
       than if we were to run our own REST API backend. At this time, this is the suggested backend.
+
+
+
+
+#Design.md by Joshua Rupp
+
+######CEG 4110 Spring 2021 
+
+######Joshua Rupp, Kyle Sturdevant, Trenton Brown --> Group 12
+
+######Overview of what you will find in this document:
+
+[Five requirements of Group 12's Google Doc which can be found at:]
+ 
+(https://docs.google.com/document/d/1LdCnvcZ_LXHHckbBnyPZtQZOH8F8qjPd47igwdDOoVs/edit)
```

Comment:
Google has permissions on files, so we may need to add Buck's account to this file if we want him to see it. Also, all of the content of this file should be moved to the Requirements.md file in the Repo. So we should not need this link at all.

Reply:
Ok, I will remove it.